### PR TITLE
initial commit -- update images and links -- DOC-2287

### PR DESCRIPTION
**Pull request by kat-enos**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*
Migration of SMP documents.

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by kat-enos** [comment]
Merging sources for the SMP docs into Developer Hub

File: docs/self-managed-enterprise-edition/self-managed-enterprise-edition-guide/how-to-use-self-signed-certificates-with-self-managed.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+volumeMounts:  
+- mountPath: /cacerts  
+  name: custom-truststore  
+  readOnly: true  
+- name: certvol  
+  mountPath: /tmp/ca.bundle  
+  subPath: ca.bundle
+```
+
+1. Locate the template `spec` and add the following volumes:
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/self-managed-enterprise-edition-guide/how-to-use-self-signed-certificates-with-self-managed.md:98]: This should be number 3.

File: docs/self-managed-enterprise-edition/self-managed-enterprise-edition-guide/virtual-machine-on-prem-backup-and-recovery.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+3. Select **Enable automatic scheduled snapshots**.
+4. In **Schedule**, select the schedule for the snapshot.
+5. In **Retention policy**, define how long to keep snapshots.
+
+The retention policy is described by KOTS:
+
+
+> The default retention period for snapshots is 1 month. Setting the retention only affects snapshots created after the time of the change. For example, if an existing snapshot had a retention of 1 year and is already 6 months old, and a user then uses the UI to change the retention to 1 month, the existing snapshot will still be around for another 6 months.
+
+1. Click **Update schedule**. The schedule is updated. To disable it, deselect **Enable automatic scheduled snapshots**.
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/self-managed-enterprise-edition-guide/virtual-machine-on-prem-backup-and-recovery.md:98]: This should be number 6.

File: docs/self-managed-enterprise-edition/sample.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/sample.md:2]: Delete this file. This was the placeholder.

File: docs/self-managed-enterprise-edition/introduction/getting-started-with-self-managed-enterprise-edition.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+title: Getting started with Self-Managed Enterprise Edition
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/introduction/getting-started-with-self-managed-enterprise-edition.md:2]:
```suggestion
title: Get started with Self-Managed Enterprise Edition
```

File: docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md [hunk 5]
```diff
@@ -0,0 +1,790 @@
...
+In this scenario, you have an existing Harness Self-Managed Enterprise Edition FirstGen cluster running and you want to add Harness Self-Managed Enterprise Edition NextGen to it.
+
+You simply add Harness Self-Managed Enterprise Edition NextGen as a new application in your existing FirstGen installation.
+
+1. Open the FirstGen KOTS admin tool.
+2. Install NextGen as a new application on existing FirstGen cluster.
+3. Upload the NextGen license file.
+4. Use the exact same FirstGen configuration values for the NextGen configuration.
+
+If you are using this option, skip to [Install NextGen on Existing FirstGen Cluster](kubernetes-cluster-on-prem-kubernetes-cluster-setup.md#install-next-gen-on-existing-first-gen-cluster).
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md:44]: This is a dead link.

File: docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md [hunk 6]
```diff
@@ -0,0 +1,790 @@
...
+1. Create a new Kubernetes cluster following the steps in the [Self-Managed Enterprise Edition - Kubernetes Cluster: Infrastructure Requirements](kubernetes-cluster-on-prem-infrastructure-requirements.md).
+2. Install FirstGen.
+3. Install NextGen as a new application on existing FirstGen cluster.
+4. Upload the NextGen license file.
+5. Use the exact same FirstGen configuration values for the NextGen configuration.
+
+If you are using this option, do the following:
+
+1. Follow all of the FirstGen installation instructions beginning with [Step 1: Set up Cluster Requirements](kubernetes-cluster-on-prem-kubernetes-cluster-setup.md#step-1-set-up-cluster-requirements).
+2. Follow the NextGen installation instructions in [Install NextGen on Existing FirstGen Cluster](kubernetes-cluster-on-prem-kubernetes-cluster-setup.md#install-next-gen-on-existing-first-gen-cluster).
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md:59]: This is a dead link.

File: docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md [hunk 7]
```diff
@@ -0,0 +1,790 @@
...
+3. Migrate data to new FirstGen using a script from Harness Support.
+4. Install NextGen as a new application on the new FirstGen cluster.
+5. Upload the NextGen license file.
+6. Use the exact same FirstGen configuration values for the NextGen configuration.
+
+If you are using this option, do the following:
+
+1. Follow all of the FirstGen installation instructions beginning with [Step 1: Set up Cluster Requirements](kubernetes-cluster-on-prem-kubernetes-cluster-setup.md#step-1-set-up-cluster-requirements).
+2. Migrate data to new FirstGen using a script from Harness Support.
+3. Follow the NextGen installation instructions in [Install NextGen on Existing FirstGen Cluster](kubernetes-cluster-on-prem-kubernetes-cluster-setup.md#install-next-gen-on-existing-first-gen-cluster).
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md:78]: This is a dead link.

File: docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md [hunk 8]
```diff
@@ -0,0 +1,790 @@
...
+
+Later, when you set up the kustomization for Harness Self-Managed Enterprise Edition, you will provide an IP address for the cluster load balancer settings.
+
+Finally, when you configure the Harness Self-Managed Enterprise Edition application, you will provide the Load Balancer URL. This URL is what Harness Self-Managed Enterprise Edition users will use.
+
+### Using NodePort?
+
+If you are creating the load balancer's Service type using NodePort, create a load balancer that points to any port in range 30000-32767 on the node pool on which the Kubernetes cluster is running.
+
+If you are using NodePort, you can skip to [Step 3: Configure Harness](#step_3_configure_harness).
```
Comment (KimberlyFields) [inline docs/self-managed-enterprise-edition/deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md:100]: This is a dead link.

**Review by KimberlyFields — CHANGES_REQUESTED** [review]
I found a few things that need to be cleaned up.

**Pull request merged**